php/Terminus/Models/Upstream.php

```diff
@@ -27,6 +27,14 @@ public function __construct($attributes, array $options = [])
         }
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function __toString()
+    {
+        return "{$this->get('id')}: {$this->get('name')}";
```

Comment:
I see why you wanted to use the URL - the name data doesn't come over in the site request and so the name is unavailable. Please change this to URL.

It's a bit picky, but would you use `$this->id` instead of `$this->get('id')` for uniformity, please?